inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.h

```diff
@@ -44,6 +44,16 @@ class MKLDNNConvolutionNode : public MKLDNNNode {
     std::vector<float> weightsZeroPoints;
     std::vector<int32_t> outputCompensation;
 
+    void setStride(const std::vector<ptrdiff_t> &_stride) { stride = _stride; }
```

Comment:
Seems like this change is not actual anymore?

Reply:
removed